test/common/upstream/load_balancer_simulation_test.cc

```diff
@@ -107,8 +107,7 @@ TEST(DISABLED_LeastRequestLoadBalancerWeightTest, Weight) {
 class DISABLED_SimulationTest : public testing::Test {
```

Comment:
I know this is not your change, but we need to mollify preexisting clang-tidy

```suggestion
class DISABLED_SimulationTest : public testing::Test { // NOLINT(readability-identifier-naming)
```

Reply:
Thanks!